docs/Supplement.qmd

```diff
@@ -0,0 +1,166 @@
+
+The multiplicative model works by iteratively "filling in" the reporting triangle starting from the bottom left, and moving column by column from left to right until the bottom right of the triangle is filled in.
+
+![Visual description of the iterative “completing” of the reporting triangle, moving from left to right and bottom to top. In this cases, we are imputing \$x\_{t=6, d = 2}\$ and \$x\_{t=5, d= 2}\$ assuming that the ratio between \$x\_{t=1:4, d = 2}\$ (block top), and \$x\_{t=1:4, d=0:1}\$ (block top left) holds for for \$x\_{t=5:6, d = 2}\$ (block bottom) and \$x\_{t=5:6, d = 0:1}\$ (block bottom left). In this example, \$\\hat{x}\_{t=6, d = 1}\$ has already been imputed using the same approach, and we treat it as known going forward. This process is repeated across the reporting triangle to estimate all values outlined in the dashed lines.](schematic_fig.png){#fig1}
+
+The method requires at least one observation, at delay $d=0$ for the most recent reference time, located at the bottom left of the reporting triangle in Figure \@ref(fig1) above.
+This means that in practice, the reporting triangle must have at least the same number of rows as it does columns.
+The method assumes that the values at each delay $d$ for the recent times, $t$, will consist of the same proportion of the values previously reported for earlier times $t$.
+To fill in the missing values for each column $d$, we sum over the rectangle of completely observed reference times for all $d-1$ columns (block top left) and sum over the column of completely observed reference times for all of the entries in column $d$ (block left).
+The ratio of these two sums is assumed to be the same in the missing entries in column $d$, so we use the entries observed up to $d-1$ for each incomplete reference time (block bottom left), and scale by this ratio to get the missing entries in column $d$.
+This process is repeated for each delay from $d$ up to the maximum delay $D$.
+At each iteration an additional reference time entry is computed as the delay $d$ increases.
+
+The delay distribution is then estimated from the filled in reporting matrix, using the same algorithm as described above for the case of the complete reporting matrix.
+
+### Point nowcast generation
+
+To "fill in" the reporting triangle from the delay distribution, we need to estimate the expected total number of eventual observed cases $\hat{x}_t$, for each reference time $t$.
+Let $z$ be the sum over all delays $d$ that have already been observed (up until $t^*-t$), such that $z =\sum_{d=1}^{d=t^*-t} x_{t,d}$ and $\delta$ be the cumulative sum of the delay distribution, $\pi(d)$ up until $d = t^*-t$ such that $\delta = \sum_{d=1}^{d=t^*-t} \pi(d)$.
+By assuming that $z \sim Bin(x_t, \delta)$ and $x_t \sim Unif(0, \inf)$, it can be shown @Cori_et_al that the expected value of $x_t$, the total number of reported cases on reference time $t$, can be written as:
```

Comment:
We discussed fleshing this out with its own SI section and getting Johannes to fill it in + invite the co-authors who developed it to be authors.